docs/nidigital/enums.rst

```diff
@@ -6,6 +6,96 @@ Enums used in NI-Digital Pattern Driver
 .. py:currentmodule:: nidigital
 
 
+ApertureTimeUnits
+-----------------
+
+.. py:class:: ApertureTimeUnits
+
+    .. py:attribute:: ApertureTimeUnits.SECONDS
+
+
+
+DigitalEdge
```

Comment:
We should double check our conventions in the nimi-python APIs as far as enum naming goes. I think we use plural for the names.

If not explicitly stated in our wiki, we should.

Reply:
From nidcpower `DigitalEdge`

I don't think we are consistent, nor do I think we can be. Some it makes sense to make plural and some it does not.